packages/tupaia-web/src/features/Map/Map.tsx

```diff
@@ -48,13 +48,14 @@ const StyledMap = styled(LeafletMap)`
     top: -50px;
     right: 3px;
     a {
-      background: rgba(43, 45, 56, 0.8);
+      background: #34353f;
       box-shadow: none;
       border: none;
       color: white;
 
       &:hover {
-        background: ${TRANSPARENT_BLACK};
+        background: ${({ theme }) =>
+          theme.palette.type === 'light' ? 'white' : 'rgba(43, 45, 56, 0.7)'};
```

Comment:
I think we don't need to worry about light theme right now. I wonder if we can make use of `lighten` from `material-ui/core` here, instead of introducing another color?